src/ImageSharp/Processing/Processors/Transforms/IResamplingImageProcessor{TPixel}.cs

```diff
@@ -0,0 +1,23 @@
+// Copyright (c) Six Labors and contributors.
+// Licensed under the Apache License, Version 2.0.
+
+using SixLabors.ImageSharp.PixelFormats;
+
+namespace SixLabors.ImageSharp.Processing.Processors.Transforms
+{
+    /// <summary>
+    /// Implements an algorithm to alter the pixels of an image via a resampling transforms.
+    /// </summary>
+    /// <typeparam name="TPixel">The pixel format.</typeparam>
+    public interface IResamplingImageProcessor<TPixel> : IImageProcessor<TPixel>
```

Comment:
`IResamplingTransformProcessor` to make it more explicit and more consistent with the `ApplyTransform` methods?

Reply:
Yeah will do.